ui/src/app/apps/apps/apps.component.html

```diff
@@ -0,0 +1,190 @@
+<div [ngBusy]="busy" id="applications-list">
+
+  <div class="heading">
+    <div class="actions" [appRoles]="['ROLE_CREATE']">
+      <div class="dropdown" dropdown id="add">
+        <button dropdownToggle type="button" class="btn btn-primary btn-dropdown" data-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="false">
+          Add Application(s)
+          <span class="caret"></span>
+        </button>
+        <ul *dropdownMenu class="dropdown-menu dropdown-menu-right">
+          <li><a (click)="registerApps()">Register application(s)</a></li>
+          <li><a (click)="bulkImportApps()">Bulk import application(s)</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="description">
+      <h1>Apps</h1>
+      <p>
+        This section lists all the available applications and provides the control to register/unregister them (if
+        applicable).
+      </p>
+    </div>
+  </div>
+
+  <div id="filters" class="pui-filter" *ngIf="!isAppsEmpty()">
+    <form #registerAppsForm="ngForm" name="registerAppsForm" role="form" (ngSubmit)="search()" novalidate class="row">
+      <div class="col-md-8">
+        <div id="dropdown-actions" class="dropdown" *ngIf="countSelected() > 0" dropdown>
+          <button dropdownToggle type="button" class="btn btn-default btn-dropdown" data-toggle="dropdown"
+                  aria-haspopup="true"
+                  aria-expanded="false">
+            Actions&nbsp;
+            <span *ngIf="countSelected() == 1">(1)</span>
+            <span *ngIf="countSelected() > 1">({{ countSelected() }})</span>
+            <span class="caret"></span>
+          </button>
+          <ul *dropdownMenu class="dropdown-menu">
+            <li><a id="unregister-apps" href="#" (click)="unregisterAppsSelected()">Unregister application(s)</a></li>
```

Comment:
Maybe add  the number of messages to unregister and pluralize accordingly.

- `Unregister 1 application`
- `Unregister 4 applications`

![image](https://user-images.githubusercontent.com/685363/36179462-78a2edc8-10c0-11e8-9cf1-01d848aba29d.png)